recipe/meta.yaml

```diff
@@ -21,10 +21,11 @@ build:
 requirements:
   host:
     - pip
-    - python >=3.6
+    - python >=3.7
     - setuptools
+    - tomli
   run:
-    - python >=3.6
+    - python >=3.7
```

Comment:
This should have been updated in 0.23 based on this upstream change ( https://github.com/python-versioneer/python-versioneer/pull/288 ).

Reply:
Hot-fixing repodata with PR ( https://github.com/conda-forge/conda-forge-repodata-patches-feedstock/pull/338 ).